app/views/saasbook/_table_of_contents.html.erb

```diff
@@ -17,6 +17,7 @@
 	<nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
     <div class="bd-toc-item active">
         <ul class="nav bd-sidenav">
+        <a class="login-button" href= "<%= user_github_omniauth_authorize_path %>">LOGIN</a>
```

Comment:
is this a temporary login until the other part is merged? aren't we putting it on the topbar, so is this necessary to merge?

Reply:
Actually that file wasn't supposed to get merged, good catch.